json_rpc/clients/httpclient.nim

```diff
@@ -220,8 +220,12 @@ proc connect*(client: RpcHttpClient, address: string, port: Port, secure: bool)
   else:
     raise newException(RpcAddressUnresolvableError, res.error)
 
+method isConnected*(client: RpcHttpClient): bool =
+  (not client.httpSession.isNil()) and client.httpAddress.isOk()
+
 method close*(client: RpcHttpClient) {.async.} =
   if not client.httpSession.isNil:
     await client.httpSession.closeWait()
+    client.httpSession = nil
```

Comment:
It appears that this change has broken the CI for some reason. I'm looking into the cause.

Reply:
testhttps.nim too 
https://github.com/status-im/nim-json-rpc/blob/9da5b609bf27805ffaa598efd2db02c1eb5019f4/tests/testhttps.nim#L91

Reply:
I see, thanks. I'll fix.